tests/subsys/fs/nffs_fs_api/basic/CMakeLists.txt

```diff
@@ -31,4 +31,9 @@ endif()
 
 zephyr_include_directories(../common)
 FILE(GLOB app_sources ../common/*.c src/*.c)
+
+if(CONFIG_BOARD_QEMU_X86)
```

Comment:
does it need to be CONFiG_BOARD_QEMU_X86? How about `CONFIG_QEMU_TARGET` instead which covers all qemus?